pkgs/top-level/all-packages.nix

```diff
@@ -30881,4 +30881,6 @@ in
   lc3tools = callPackage ../development/tools/lc3tools {};
 
   zktree = callPackage ../applications/misc/zktree {};
+
+  ignite = callPackage ../applications/virtualization/ignite {};
```

Comment:
Please sort this entry. Adding it to the end of the list creates merge conflicts if everyone does this.